packages/language-server/src/model/model-builder.ts

```diff
@@ -211,6 +212,32 @@ function buildModel(services: LikeC4Services, docs: ParsedLikeC4LangiumDocument[
     indexBy(prop('id'))
   )
 
+  const parsedGlobalRules: {
+    styles: Record<c4.GlobalStyleID, c4.ViewRuleStyle[]>,
+  } = {
+    styles: {},
+  }
+  Object.assign(parsedGlobalRules.styles, ...docs.map(d => d.c4Global.styles))
+
+  const globalRules: {
+    styles: Record<c4.GlobalStyleID, c4.GlobalStyle>,
+  } = {
+    styles: {},
+  }
+
+  for (const key in parsedGlobalRules.styles) {
```

Comment:
I suggest replace with 
```suggestion
  for (const [id, styles] of entries(parsedGlobalRules.styles)) {
```
to remove redundant `if` clause in `for`

or even
```ts
globalRules.styles = pipe(
  entries(parsedGlobalRules.styles),
  map(([id,style]) => ({id, style})),
  indexBy(prop('id'))
)
```

Reply:
Thanks for this solution! I spent long time trying to figure out how to do this in TypeScript. Finally I gave up.

I'll get familiar with this library. Now I have no clue how it works. But the tests are passing :)